manage/roles/agent-permissions.rst

```diff
@@ -40,6 +47,26 @@ Agent Permissions
                                                             .. hint::
 
                                                                Public articles are always visible.
+
+                            .. tip::
+
+                               Zammad supports `granular permissions on knowledge base categories`_.
+
+                               This function allows *agents with editor* permissions to restrict
+                               roles to specific internal categories & answers.
```

Comment:
This function allows *agents with editor permissions* to restrict specific internal categories & answers to chosen roles.